src/twisted/web/http.py

```diff
@@ -1895,12 +1894,21 @@ class _ChunkedTransferDecoder:
         state transition this is truncated at the front so that index 0 is
         where the next state shall begin.
 
-    @ivar _start: While in the C{'CHUNK_LENGTH'} state, tracks the index into
-        the buffer at which search for CRLF should resume. Resuming the search
-        at this position avoids doing quadratic work if the chunk length line
-        arrives over many calls to C{dataReceived}.
+    @ivar _start: While in the C{'CHUNK_LENGTH'} and C{'TRAILER'} states,
+        tracks the index into the buffer at which search for CRLF should resume.
+        Resuming the search at this position avoids doing quadratic work if the
+        chunk length line arrives over many calls to C{dataReceived}. Not used
+        in any other state.
```

Comment:
Very minor comment.

Maybe other states will use it and forget to update the docstring.

So maybe this is better :) 
```suggestion
        chunk length line arrives over many calls to C{dataReceived}.
```